GVFS/GVFS.Platform.Mac/MacFileSystemVirtualizer.cs

```diff
@@ -32,6 +34,14 @@ public MacFileSystemVirtualizer(GVFSContext context, GVFSGitObjects gitObjects)
             this.virtualizationInstance = virtualizationInstance ?? new VirtualizationInstance();
         }
 
+        protected override string EtwArea

```

Comment:
Nit pick: this can be rewritten as `protected override string EtwArea => ClassName;` and avoid taking up so much screen space for boilerplate.